### PR TITLE
Matplotlib style and plot size for latex

### PR DESCRIPTION
**Pull request by mturilli**
This add a Matplotlib style and a function to calculate the size of a plot for a latex document. Workflow is documented in plotting.rst.

### CODE REVIEW

File: src/radical/analytics/__init__.py [hunk 1]
```diff
@@ -24,6 +28,26 @@
 logger = ru.Logger('radical.analytics')
 logger.info('radical.analytics    version: %s' % version_detail)
 
+
 # ------------------------------------------------------------------------------
+#
```
Comment (andre-merzky) [inline src/radical/analytics/__init__.py:33]: Shouldn't this move to `utils/plot.py` also?  Also, I would suggest to change the names to `get_plotsize()` and `get_plotstyle()` to make the purpose clearer.

File: src/radical/analytics/utils/__init__.py [hunk 2]
```diff
@@ -0,0 +1,3 @@
+# ------------------------------------------------------------------------------
+#
+from .plot import *
```
Comment (andre-merzky) [inline src/radical/analytics/utils/__init__.py:3]: Pylint wants us to use the explicit specifiers, thus `from .plot import get_size` etc.  Also, please do add a newline.

**Review by andre-merzky — CHANGES_REQUESTED** [review]

File: src/radical/analytics/__init__.py (hunk 1 above)
Comment (mturilli) [reply src/radical/analytics/__init__.py:33]: Done.

File: src/radical/analytics/utils/__init__.py (hunk 2 above)
Comment (mturilli) [reply src/radical/analytics/utils/__init__.py:3]: Done.

**Review by andre-merzky — APPROVED** [review]
Thanks, LGTM

**Pull request merged**